### PR TITLE
7  config endpoints (master)

### PR DESCRIPTION
**Pull request by jasoncoposky**

### CODE REVIEW

**Comment by trel** [comment]
this PR has connection pool commit also - can/will go away after rebasing atop other (auth) PR

File: api/QueryApi.cpp [hunk 1]
```diff
@@ -51,13 +51,11 @@ void QueryApi::setupRoutes() {
...
-    auto queryLimit = request.query().get("query_limit");
-    auto rowOffset = request.query().get("row_offset");
-    auto queryType = request.query().get("query_type");
-    
-      this->catalog_query(request.headers(), request.body(), queryString, queryType, queryLimit, rowOffset, response);
+        auto queryLimit = request.query().get("query_limit");
+        auto rowOffset  = request.query().get("row_offset");
+        auto queryType  = request.query().get("query_type");
+
+        this->catalog_query(request.headers(), request.body(), queryString, queryType, queryLimit, rowOffset, response);
```
[the remark this answers is not in the document]
Comment (jasoncoposky) [reply api/QueryApi.cpp:58]: it is there in the file in my branch for this PR?

[the remark this answers is not in the document]
Comment (trel) [reply api/QueryApi.cpp:58]: okay, let's get the auth PR merged, and then we'll revisit this one - something is afoot

[the remark this answers is not in the document]
Comment (trel) [reply api/QueryApi.cpp:58]: yep, now with more merge conflict!

**Comment by trel** [comment]
Are we missing the same update here as in 4-2-stable?

irods_specific_implementation/CMakeLists.txt:
```diff
-                        /opt/irods-externals/pistache0.0.2-0/include
+                        ${IRODS_EXTERNALS_FULLPATH_PISTACHE}/include
```

**Pull request merged**